package.json

```diff
@@ -9,6 +9,7 @@
 		"dev": "nx dev playground-website",
 		"dev:interactivity": "nx dev interactive-block-playground",
 		"format": "nx format",
+		"format:uncommitted": "nx format --fix --parallel --uncommitted",
```

Comment:
I _think_ this does what I want it to, but I want to make sure this only formats _staged_ files and not all uncommitted files, because if I have unstaged changes and untracked files I don't want the formatter to touch those

Reply:
`git commit -n` it is then…